packages/scripts/src/types/state.ts

```diff
@@ -0,0 +1,10 @@
+// export const enum ReadyState { loading = 1, failed, ready }
+export enum ReadyState { loading = 'loading', failed = 'failed', ready = 'ready' }
+
+export interface State{
+  scripts: any[]
+  scriptTypes: string[]
```

Comment:
could fix this to the types we have

Reply:
Rebuilding the complete API structure into a complex datatype feels like a lot of over engineering just to remove the "any". I think that a more 'save' type will also take a lot of time then something about the api changes. (like adding a new field)

Reply:
No changes on the api response do not necessarily directly impact the model, if they do then it would be good to know about it. I do not suggest you 'Rebuilding the complete API structure', i suggest to add type information to the  frontend model, to communicate the model structure to other devs ( or your future self :) )